src/test/java/org/apache/datasketches/memory/test/MemoryTest.java

```diff
@@ -383,17 +430,17 @@ public void checkMonitorDirectStats() {
     int bytes = 1024;
     WritableHandle wh1 = WritableMemory.allocateDirect(bytes);
     WritableHandle wh2 = WritableMemory.allocateDirect(bytes);
-    assertEquals(BaseState.getCurrentDirectMemoryAllocations(), 2L);
-    assertEquals(BaseState.getCurrentDirectMemoryAllocated(), 2 * bytes);
+    assertEquals(getCurrentDirectMemoryAllocations(), 2L);
+    assertEquals(getCurrentDirectMemoryAllocated(), 2 * bytes);
 
     wh1.close();
-    assertEquals(BaseState.getCurrentDirectMemoryAllocations(), 1L);
-    assertEquals(BaseState.getCurrentDirectMemoryAllocated(), bytes);
+    assertEquals(getCurrentDirectMemoryAllocations(), 1L);
+    assertEquals(getCurrentDirectMemoryAllocated(), bytes);
 
     wh2.close();
     wh2.close(); //check that it doesn't go negative.
-    assertEquals(BaseState.getCurrentDirectMemoryAllocations(), 0L);
-    assertEquals(BaseState.getCurrentDirectMemoryAllocated(), 0L);
+    assertEquals(getCurrentDirectMemoryAllocations(), 0L);
+    assertEquals(getCurrentDirectMemoryAllocated(), 0L);
```

Comment:
Do you allocate the direct memory through the public API and make assertions about the state number of allocations and allocated bytes through reflection?  This is an important distinction that I need to understand - as earlier in the PR you made many static methods public.

Reply:
Thanks for the detailed explanation - I now understand the process for how the changes were introduced, and confirms my understanding is correct.